tests/config/Issue_2795.cfg

```diff
@@ -0,0 +1 @@
+indent_columns = 4
```

Comment:
Based on the problem description I'd expected a config containing the `code_width` option. What is tested here? 

Also please keep the naming scheme: It should be named indent_columns-4.cfg so it can be reused for other tests. There is nothing gained from the issue ticket here, you have that info already in the input file name: ` cpp/Issue_2795.cpp`.

Reply:
Thanks for the comments